### PR TITLE
Queries on Manual Setup doc

### PR DESCRIPTION
**Pull request by neiljerram**
Proposing as a starting point for discussion - not an actual change as it currently stands.

### CODE REVIEW

File: docs/calico-with-docker/without-docker-networking/ManualSetup.md [hunk 1]
```diff
@@ -28,7 +28,13 @@ They must have the following software installed:
 
 - Docker v1.6 or greater: [Docker][docker]
 - etcd installed and available on each node: [etcd documentation][etcd]
+
+[NJ] What does 'available' mean?  Does this mean that I need a configured and active etcd system, such that I can do 'etcdctl ls /' on each node?  Any recommendations or restrictions on how many etcd masters, or where I can use proxies?
+
```
Comment (fasaxc) [inline docs/calico-with-docker/without-docker-networking/ManualSetup.md]: At this stage, we can support most (all?) etcd topologies, with or without proxies.  If you're running a local proxy then we work without config.  If not, you need to configure the correct URL(s) for us to connect to.

**Comment by matthewdupre** [comment]
I think the first of these is fine as it is (see Shaun's comment), and the second we've changed elsewhere.  Closing.

**Comment by neiljerram** [comment]
@matthewdupre @fasaxc I think I didn't express my first point well.  Imagine a new reader who is going through the process step by step.  They see an instruction saying that etcd must be installed - and that's fine, because everyone knows what 'installed' means.  But 'and available'?  Huh, why does it also say that; what does it mean?

I think the answer is nothing, i.e. this is just verbosity - because required etcd setup is in fact covered more properly, just a few lines further below.  I'll propose another PR to clean this up.

**Comment by neiljerram** [comment]
New PR is at https://github.com/projectcalico/calico-containers/pull/1011

**Pull request closed without merging**